src/client/dfuse/il/int_posix.c

```diff
@@ -2493,7 +2493,7 @@ dfuse_fgets(char *str, int n, FILE *stream)
 }
 
 DFUSE_PUBLIC wchar_t *
-dfuse_fgetws(const wchar_t *ws, FILE *stream)
+dfuse_fgetws(const wchar_t *ws, int n, FILE *stream)
```

Comment:
Can you remove the D_ERROR call on L2502 as well please.

Reply:
Done in next commit